build/configs/rtl8730e/loadable_apps_16M/defconfig

```diff
@@ -299,7 +299,22 @@ CONFIG_AUTOMOUNT_USERFS=y
 #
 # SE Selection
 #
-# CONFIG_SE is not set
+CONFIG_SE=y
+# CONFIG_DEBUG_SECURE_ELEMENT_ERROR is not set
+# CONFIG_SE_SSS is not set
+# CONFIG_SE_KONAI is not set
+# CONFIG_SE_UONEA is not set
+# CONFIG_SE_VIRTUAL is not set
+CONFIG_SE_AMEBA=y
+CONFIG_HW_RNG=y
+CONFIG_HW_DH_PARAM=y
+CONFIG_HW_ECDH_PARAM=y
+CONFIG_HW_RSA_VERIFICATION=y
+CONFIG_HW_ECDSA_VERIFICATION=y
+CONFIG_HW_RSA_ENC=y
+CONFIG_HW_SE_STORAGE=y
+# CONFIG_HW_AES_ENC is not set
+CONFIG_SE_SECURE_CONTEXT_SIZE=6144
```

Comment:
Please check and apply the value of the hello.

Reply:
SECURE CONTEXT SIZE is not used in rtl8730e. But I changed the value for consistency.